### PR TITLE
Add translation for OTP concurrency - Portuguese

### PR DESCRIPTION
**Pull request by philss**
Related to #122 

Note: I couldn't find a good translation for `callback`, so I leave as it is but in *italic*.
Note 2: The word `handler` is not easy to translate, so I used the word "processador" which is something similar.

### CODE REVIEW

**Comment by philss** [comment]
@pragmaticivan can you review this one?

File: pt/lessons/advanced/otp-concurrency.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+Para mais informações olhe a documentação oficial do [GenServer](http://elixir-lang.org/docs/v1.1/elixir/GenServer.html#content).
+
+## GenEvent
+
+Nós aprendemos que GenServers são processos que podem manter estado e processar requisições síncronas e assíncronas, então o que é um GenEvent? GenEvents são administradores de eventos genéricos que recebem eventos e notificam consumidores inscritos. Eles promovem um mecanismo para adicionar ou remover dinamicamente "processadores" ao fluxo de eventos.
+
+### Processando eventos
+
+O mais importante *callback* em GenEvents, como você pode imaginar, é o `handle_event/2`. Ele recebe o evento e o estado atual do processador e é esperado o retorno de uma tupla: `{:ok, estado}`.
```
Comment (pragmaticivan) [inline pt/lessons/advanced/otp-concurrency.md]: I think would be better `O *callback* mais importante em GenEvents`

File: pt/lessons/advanced/otp-concurrency.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+Para demonstrar a funcionadlidade do GenEvent, vamos começar por criar dois processadores, um para manter um log de mensagens e outro para persistir elas (teoricamente):
```
Comment (pragmaticivan) [inline pt/lessons/advanced/otp-concurrency.md]: I think would be better `vamos começar criando dois processadores`

**Comment by pragmaticivan** [comment]
@philss Just left some comments. Thanks! :beers:

File: pt/lessons/advanced/otp-concurrency.md (hunk 1 above)
Comment (philss) [inline pt/lessons/advanced/otp-concurrency.md]: 4205b16

File: pt/lessons/advanced/otp-concurrency.md (hunk 2 above)
Comment (philss) [inline pt/lessons/advanced/otp-concurrency.md]: 4205b16

**Comment by philss** [comment]
@pragmaticivan thank you! fixed :beers:

**Comment by pragmaticivan** [comment]
Nice, thanks :sparkles: o/

**Pull request merged**